### PR TITLE
Fix Bug 1124724 - Tiles product page update copy mozilla.org/en-US/firefox/tiles/

### PR DESCRIPTION
**Pull request by kyoshino**
@flodolo does this work for you?

### CODE REVIEW

File: bedrock/firefox/templates/firefox/tiles.html [hunk 1]
```diff
@@ -14,8 +14,15 @@
    3. Breadcrumbs ("Home > Title")
    3. Page title (large headline)
 #}
-{% block page_title %}{{ _('A Brand New Tiles Experience') }}{% endblock %}
 
+{% block page_title -%}
+{%- if l10n_has_tag('bug_1124724_copy_update') -%}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/tiles.html]: No need to have a tag this detailed, `tiles_update` is enough. Also, the comment is not necessary (`L10n: This sentence is obsolete. Do not translate.`).

**Comment by flodolo** [comment]
I find the template almost impossible to decipher, there is so much duplicated code that I can't really understand how much text is changed.

Example: both code sections have a ```What are Enhanced Tiles``` section starting with `h3`. The ```if l10n_has_tag``` should be inside that section for the updated text

**Comment by kyoshino** [comment]
Sorry for the trouble :bow: Fixed the readability and comments.

**Comment by flodolo** [comment]
Per discussion on the bug, the new content actually needs to be en-US only.

**Comment by kyoshino** [comment]
Got it :+1:

**Comment by alexgibson** [comment]
Just checking in to review here - @flodolo so you recommend removing the use of `l10n_has_tag('tiles_update')` and making these string changes en-US only for now? Discussion in the bug seems to suggest this copy is all changing again in late Q1 anyway, so this makes sense to me.

@kyoshino does this sound ok to you to update?

**Comment by flodolo** [comment]
Yes, remove the tag and put these strings behind a ```if en-US``` cycle like [this one](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/tiles.html#L64-L74).

**Comment by kyoshino** [comment]
Replaced `{% if l10n_has_tag('tiles_update') %}` with `{% if LANG in ['en-US'] %}`.

**Comment by kyoshino** [comment]
I'll unwrap the en-US strings later.

**Comment by kyoshino** [comment]
Done unwrapping (so far).

**Comment by alexgibson** [comment]
Thanks @kyoshino, looks good to me r+

**Pull request merged**